### ISSUE TITLE
Style content gets escaped

### ISSUE THREAD

**Issue by Miha-x64** [Kotlin/kotlinx.html#58]
```kt
style(content = "ul > li { ... }")
```
will be rendered like
```html
<style>
ul &gt; li { ... }
</style>
```
which is wrong. Using
```kt
style {
    unsafe { +" ... "}
}
```
as a workaround at the moment.

**Comment by cy6erGn0m** [comment]
`content` attribute should be deprecated for tags that usually require unsafe content inside

**Comment by cy6erGn0m** [comment]
Fixed in 0.6.5
